extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -286,6 +291,13 @@ export class ConsentUI {
    * @return {!Promise}
    */
   loadIframe_() {
+    const info = dict({});
+    if (this.inlineSetting_) {
+      info['setting'] = this.inlineSetting_;
+    }
+
+    this.ui_.setAttribute('name', JSON.stringify(info));
```

Comment:
If I remember correctly, you will need to pass a sentinel (as this is what we do to communicate with 3p frames), and stringify using a `dict()`. 

See: https://github.com/ampproject/amphtml/blob/master/extensions/amp-recaptcha-input/0.1/amp-recaptcha-service.js#L222

Reply:
Discussed offline, don't need a sentinel for this use case, and dict was used three lines above 🙃